server/monitor/pom.xml

```diff
@@ -27,6 +27,11 @@
   <name>Apache Accumulo Monitor Server</name>
   <description>A web server for monitoring Apache Accumulo.</description>
   <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
```

Comment:
versions should be managed in the parent POM's dependencyManagement section; also, if this is a new dependency, does it also need to be included in the assembly?

Reply:
No worries there, its already in the assemble pom.  Will remove the version here though.